docs/content/en/schemas/v2beta7.json

```diff
@@ -1639,15 +1639,45 @@
           "description": "a directory to read functions from instead of the configuration directory.",
           "x-intellij-html-description": "a directory to read functions from instead of the configuration directory."
         },
+        "globalScope": {
+          "type": "boolean",
+          "description": "sets global scope for functions.",
+          "x-intellij-html-description": "sets global scope for functions.",
+          "default": "false"
+        },
         "image": {
           "type": "string",
           "description": "an image to be run as a function in lieu of running functions from a directory.",
           "x-intellij-html-description": "an image to be run as a function in lieu of running functions from a directory."
+        },
+        "mount": {
+          "items": {
+            "type": "string"
+          },
+          "type": "array",
+          "description": "a list of storage options to mount to the fn image.",
+          "x-intellij-html-description": "a list of storage options to mount to the fn image.",
+          "default": "[]"
+        },
+        "network": {
+          "type": "boolean",
+          "description": "enables network access for functions that declare it.",
+          "x-intellij-html-description": "enables network access for functions that declare it.",
+          "default": "false"
+        },
+        "networkName": {
+          "type": "string",
+          "description": "docker network to run the container in (default \"bridge\").",
+          "x-intellij-html-description": "docker network to run the container in (default &quot;bridge&quot;)."
         }
       },
       "preferredOrder": [
         "fnPath",
-        "image"
+        "image",
```

Comment:
nit: always use alphabetical order in config list especially if the list is large. (if some fields are required, required should always come first).

Reply:
Thanks for the tip!